app/views/widgets/thermometer/_form.slim

```diff
@@ -0,0 +1,12 @@
+#thermometer_widget
+  div.form-group
+    label for='widgets[thermometer][goal]' Goal: 
+    input type='number' name='widgets[thermometer][goal]' value=10000
+  div.form-group
+    label for='widgets[thermometer][goal]' Starting count: 
+    input type='number' name='widgets[thermometer][count]' value=10000
```

Comment:
This is one of the things that are different on the new branch - on the newer branch, this goes to templates_helper and sets the goal and starting count values from default options, which are count = 0 and goal = 10000, so this would get changed with merging the bigger PR. Is that OK or should I make the change here, as well?